src/jit/codegenlegacy.cpp

```diff
@@ -21307,6 +21310,21 @@ regNumber CodeGen::genPInvokeCallProlog(LclVarDsc*            frameListRoot,
 
     noway_assert(compiler->lvaInlinedPInvokeFrameVar != BAD_VAR_NUM);
 
+#ifdef _ARM_
+    if (compiler->opts.ShouldUsePInvokeHelpers())
+    {
+        regNumber baseReg;
+        int       adr = compiler->lvaFrameAddress(compiler->lvaInlinedPInvokeFrameVar, true, &baseReg, 0);
```

Comment:
@jkotas  as I see there the address of ```lvaInlinedPInvokeFrameVar`` is needed as argument
I'm not so good in arm arch yet.

the generated code for ARM:
```
Instruction list before instruction issue:

G_M176_IG01:        ; func=00, offs=000000H, size=000AH, gcrefRegs=0000 {}, byrefRegs=0000 {}, byref, nogc <-- Prolog IG

000000      push    {r4,r5,r6,r7,r8,r9,r10,r11,lr}
000004      sub     sp, 28
000006      add     r11, sp, 56

G_M176_IG02:        ; offs=00000AH, size=001AH, gcrefRegs=0000 {}, byrefRegs=0000 {}, byref

00000A      sub     r0, r11, 68
00000E      movw    r3, LOW RELOC 0x422eb8
000012      movt    r3, HIGH RELOC 0x422eb8
000016      blx     r3		// CORINFO_HELP_JIT_PINVOKE_BEGIN
000018      movw    r3, LOW RELOC 0x422ec0
00001C      movt    r3, HIGH RELOC 0x422ec0
000020      ldr     r3, [r3]
000022      blx     r3		// System.Runtime.InternalCalls:RhpSignalFinalizationComplete()

G_M176_IG03:        ; offs=000024H, size=0010H, gcrefRegs=0000 {}, byrefRegs=0000 {}, byref, isz

000024      sub     r0, r11, 68
000028      movw    r3, LOW RELOC 0x422ec8
00002C      movt    r3, HIGH RELOC 0x422ec8
000030      blx     r3		// CORINFO_HELP_JIT_PINVOKE_END
000032      b       SHORT G_M176_IG04

G_M176_IG04:        ; offs=000034H, size=0006H, gcrefRegs=0000 {}, byrefRegs=0000 {}, byref, epilog, nogc

000034      add     sp, 28
000036      pop     {r4,r5,r6,r7,r8,r9,r10,r11,pc}
```

The generated code for x64 is:

```
Instruction list before instruction issue:

G_M179_IG01:        ; func=00, offs=000000H, size=0013H, gcrefRegs=00000000 {}, byrefRegs=00000000 {}, byref, nogc <-- Prolog IG

000000 push     rbp
000001 push     r15
000003 push     r14
000005 push     r13
000007 push     r12
000009 push     rbx
00000A sub      rsp, 40
00000E lea      rbp, [rsp+50H]

G_M179_IG02:        ; offs=000013H, size=0009H, gcrefRegs=00000000 {}, byrefRegs=00000000 {}, byref

000013 lea      rdi, bword ptr [V00 rbp-50H]
000017 call     CORINFO_HELP_JIT_PINVOKE_BEGIN

G_M179_IG03:        ; offs=00001CH, size=0010H, gcrefRegs=00000000 {}, byrefRegs=00000000 {}, byref, isz

00001C call     System.Runtime.InternalCalls:RhpSignalFinalizationComplete()
000021 lea      rdi, bword ptr [V00 rbp-50H]
000025 call     CORINFO_HELP_JIT_PINVOKE_END
00002A jmp      SHORT G_M179_IG04

G_M179_IG04:        ; offs=00002CH, size=000FH, gcrefRegs=00000000 {}, byrefRegs=00000000 {}, byref, epilog, nogc

00002C lea      rsp, [rbp-28H]
000030 pop      rbx
000031 pop      r12
000033 pop      r13
000035 pop      r14
000037 pop      r15
000039 pop      rbp
00003A ret      
```